### PR TITLE
fix plotly import

### LINKED ISSUE #50: broken SSR
In the [SSR-Tutorial](https://design.geovistory.org/?path=/story/contributing-tutorial-server-side-rendering--server-side-rendering) we get to use the `renderToString`-Function of the web/hydrate-Package. This function broke between versions 1.4.3 and latest (1.10.0).
As it is now, the function produces a `Hydrate Error`. It tells us the following:
```
ReferenceError: DOMParser is not defined
```

I tried to take a look at the code in question, but couldn't find it in the repo. So this is all I can contribute to debugging this.

### PR DESCRIPTION
**Pull request by joschne**
This PR changes the way of importing Plotly to components: Load Plotly from CDN in browsers, don't load it on server side. This fixes server side rendering, that was broken due to Plotly and resolves #50.

In addition the PR adds unit tests using/testing the server side rendering (hydrate-script) for the components geov-entity-label (without Plotly) and geov-class-distri (with Plotly), in order to test the issue with broken SSR directly in the design-system.

In addition the PR pumps stencil core version and some dependencies. This allowed to remove a patch applied to a previous stencil version for SSR.

In addition the PR removes the unused output target 'happyDom' in order to simplify the code base.

### CODE REVIEW

**Review by perrauda — APPROVED** [review]
Review ok but this file list may not be needed in the commit:
.vscode/launch.json
LICENSE
CHANGELOG.md
README.md
package-lock.json
maybe packages/design-system-web/src/index.html (as it's used for tests, it may interfere with another test, at the same time it may be useful... I'm not sure.)

**Comment by joschne** [comment]
> Review ok but this file list may not be needed in the commit: .vscode/launch.json LICENSE CHANGELOG.md README.md package-lock.json maybe packages/design-system-web/src/index.html (as it's used for tests, it may interfere with another test, at the same time it may be useful... I'm not sure.)

- .vscode/launch.json: I agree, this can be deleted, but I can also be helpful and does no harm.
- LICENSE: was deleted as part of happyDom. Must stay in the commit.
- README.md: dito
- CHANGELOG.md: I agree, we have an issue. As long as we don't use conventional commit messages, there is nothing useful in the changelogs.
- package-lock.json is needed! see e.g.: [this post](https://dev.to/adamklein/package-lock-json-in-git-or-not-50l5)
- packages/design-system-web/src/index.html is not used for tests but for development. It can be removed and added to .gitignore, but only under the condition that we provide an example file in the dev-guide in storybook, explaining how to use this file for faster hot reloading during development. I created an issue for this: #59. 


I rather see a problem with `packages/design-system-web/src/components.d.ts` since this is auto-generated. I think it could be git-ignored, since our CI/CD pipeline will build it before publication.

File: packages/design-system-web/src/lib/loadPlotlyBasic.tsx
Comment (joschne) [inline packages/design-system-web/src/lib/loadPlotlyBasic.tsx:1]: File should be renamed to importPlotlyBasic.tsx, as this is the name of the exported function.

File: packages/design-system-web/package.json [hunk 1]
```diff
@@ -48,42 +48,41 @@
...
     "========================== STORYBOOK =========================": "",
     "start:storybook": "start-storybook -p 6006",
     "storybook": "start-storybook -p 6006",
     "build-storybook": "build-storybook"
   },
   "dependencies": {
-    "@geovistory/happy-dom-output-target": "^1.0.1-3",
-    "@stencil/core": "^2.17.4",
+    "@stencil/core": "^3.3.0",
     "@types/plotly.js": "^2.12.13",
```
Comment (joschne) [inline packages/design-system-web/package.json:60]: remove this. it's dev dependency.

File: packages/design-system-web/package.json [hunk 2]
```diff
@@ -48,42 +48,41 @@
...
-    "@storybook/addon-interactions": "^6.5.9",
-    "@storybook/addon-links": "^6.5.9",
+    "@storybook/addon-interactions": "^6.5.16",
+    "@storybook/addon-links": "^6.5.16",
     "@storybook/builder-webpack4": "^6.5.9",
     "@storybook/manager-webpack4": "^6.5.9",
     "@storybook/react": "^6.5.9",
     "@storybook/testing-library": "^0.0.13",
     "@types/jest": "^27.0.3",
+    "@types/plotly.js": "2.12.18",
```
Comment (joschne) [inline packages/design-system-web/package.json:85]: use version 2.23.2 if possible, as this version is actually loaded.

File: packages/design-system-web/src/index.html
Comment (joschne) [inline packages/design-system-web/src/index.html:1]: this file should be git-ignored: see #59 
For now, it's good

File: packages/design-system-web/src/lib/loadPlotlyBasic.tsx
Comment (joschne) [inline packages/design-system-web/src/lib/loadPlotlyBasic.tsx:1]: rename file as function

File: packages/design-system-web/package.json (hunk 2 above)
Comment (joschne) [reply packages/design-system-web/package.json:85]: latest version available at time of writing: 2.12.18

**Pull request merged**